pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -6234,7 +6243,18 @@ window_spec_opt:
     }
 
 function_call_aggregate:
-    AVG '(' func_type_opt expression  ')' window_spec_opt
+    GROUP_CONCAT '(' func_type_opt expression_list separator_opt ')' window_spec_opt
```

Comment:
remove separator_opt，just expression_list is ok, expression include string. and add unit test

Reply:
> remove separator_opt，just expression_list is ok, expression include string. and add unit test

mysql group_concat() has **SEPARATOR** keyword